packages/core/redux/session/selectors/session.js

```diff
@@ -90,5 +94,42 @@ export const getAccountProfileSummary = createSelector(
     }
 
     return {}
-  }
+  },
+)
+
+export const getAccountAddresses = createSelector(
+  [
+    getPersistAccount,
+    getEthereumSigner,
+    getBitcoinSigner,
+    getBitcoinCashSigner,
+    getLitecoinSigner,
+    getNemSigner,
+    getWavesSigner,
+  ],
+  (persistAccount, ethereumSigner, bitcoinSigner, bitcoinCashSigner, litecoinSigner, nemSigner, wavesSigner) => {
+    const { selectedWallet } = persistAccount
+    const signerPath = selectedWallet.encrypted[0].path
```

Comment:
Is there a selector for the signer path in persistAccount/selectors? We do not need whole persistAccount here.